### DISCUSSION TITLE
Big problem after use

### DISCUSSION THREAD

**Discussion by Krasnomakov** [usnistgov/macos_security#342] [Q&A]
I followed this guide 
https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-219.pdf

And performed scans. After scans I fixed issues with the tool and improved compliance percentage from 13% to 45%

However, two major problems occured. 

One is that my admin user was removed from sudoers. And I cannot use my password with sudo. 
And I cannot change sudoers as well and add my user in there. 

Another is this window that appears on restart. 
![IMG_8268](https://github.com/usnistgov/macos_security/assets/104280018/d6c38650-c5a6-45c7-9cad-f1ed37f69428)

Any suggestions? How can I acquire my admin rights back? 
And does my laptop really belongs to US government? And all my data was stolen? 

How to disable this window? And should I erase the mac now?

**Comment by Krasnomakov** [comment]
Ok, I had to reinstall OS because messed up sudoers file trying to get my admin rights back. 
Now I can use sudo. 

However, even after reinstallation I still have this agreement window on start. 

It is very disapposinting. 

Who could ever thought that my computer with all its data would turn into US government property after security improvement? 

How can I get rid of it and what does it mean? 

Please, explain.

**Comment by Krasnomakov** [comment]


**Reply by jmahlman** [reply]
I think you really need to understand what you are running on your system before running it. Everything you described is expected. I **highly** suggest you read up on what the project is, who it's for, and what the controls do. There is a lot of information out there from Apple and third party sources to aid you with this. 

This project is designed to secure macOS systems as scale for organizations that have to meet regulatory compliance. It uses existing macOS controls to harden a system to meet those requirements. All of the code is accessible and extensively documented within the GitHub page. 

A bit of advice, if you don't understand what you are running, don't run it.
